### PR TITLE
Add content/pt/docs/tutorials/kubernetes-basics/deploy-app

### PR DESCRIPTION
**Pull request by femrtnz**
related to #13939

### CODE REVIEW

**Comment by femrtnz** [comment]
Hey @devlware, can you review this one?
thanks!

File: content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+---
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+<link href="/docs/tutorials/kubernetes-basics/public/css/overrides.css" rel="stylesheet">
```
Comment (sftim) [inline content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html:13]: Should these be links to `/pt/*` ?

File: content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            </div>
+        </div>
+
+        <br>
+        <div class="katacoda">
+            <div class="katacoda__alert">
+                Para interagir com o Terminal, use a versão desktop/tablet
+            </div>
+
+            <div class="katacoda__box" id="inline-terminal-1"  data-katacoda-id="kubernetes-bootcamp/7" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;">
```
Comment (sftim) [inline content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html:34]: Optionally, localize “Kubernetes Bootcamp Terminal”

File: content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+        <div class="row">
+            <div class="col-md-8">
+                <h2 style="color: #3771e3;">Implantar seu primeiro aplicativo no Kubernetes</h2>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p><img src="/docs/tutorials/kubernetes-basics/public/images/module_02_first_app.svg"></p>
```
Comment (sftim) [inline content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:71]: Should this be `/pt/docs/tutorials/kubernetes-basics/public/images/module_02_first_app.svg` (I think so)

**Review by sftim — COMMENTED** [review]
Thanks for this. I have some suggestions about keeping this localization separate from the English upstream.

File: content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html (hunk 1 above)
Comment (femrtnz) [reply content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html:13]: I can see all other localizations are not specifying the locale on the link reference

i.e [here](https://github.com/kubernetes/website/blob/master/content/fr/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html#L12) and [here](https://github.com/kubernetes/website/blob/master/content/es/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html#L12)

File: content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html (hunk 3 above)
Comment (femrtnz) [reply content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:71]: I don't believe we have the images localized yet as well. Do you think we should add the public folder with the images event if we are not translating those?

File: content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html (hunk 1 above)
Comment (femrtnz) [reply content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html:13]: do you think we should start localizing those even if are duplicated and with the risk of stale it?

**Comment by devlware** [comment]
Hey @femrtnz!
Sorry, I was disconnected from the matrix... :)
I'll review it today.

File: content/pt/docs/tutorials/kubernetes-basics/_index.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+  <link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+<div class="layout" id="top">
+
+  <main class="content">
+
+    <div class="row">
+      <div class="col-md-9">
+        <h2>Basico do Kubernetes</h2>
```
Comment (devlware) [inline content/pt/docs/tutorials/kubernetes-basics/_index.html:26]:
```suggestion
        <h2>Básico do Kubernetes</h2>
```

File: content/pt/docs/tutorials/kubernetes-basics/_index.html [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+      </div>
+    </div>
+
+    <br>
+
+    <div class="row">
+      <div class="col-md-9">
+        <h2>O que o Kubernetes pode fazer por você?</h2>
+        <p>Com os serviços da Web modernos, os usuários esperam que os aplicativos estejam disponíveis 24 horas por dia, 7 dias por semana, e os desenvolvedores esperam implantar novas versões desses aplicativos várias vezes ao dia. A conteinerização ajuda a empacotar o software para atender a esses objetivos, permitindo que os aplicativos sejam lançados e atualizados de maneira fácil e rápida, sem tempo de inatividade. O Kubernetes ajuda a garantir que esses aplicativos em contêiner sejam executados onde e quando você quiser e os ajuda a encontrar os recursos e ferramentas de que precisam para funcionar. Kubernetes é uma plataforma de código aberto pronta para produção, projetada com a experiência acumulada do Google em orquestração de contêineres, combinada com as melhores ideias da comunidade.</p>
```
Comment (devlware) [inline content/pt/docs/tutorials/kubernetes-basics/_index.html:44]:
```suggestion
        <p>Com os serviços da Web modernos, os usuários esperam que os aplicativos estejam disponíveis 24 horas por dia, 7 dias por semana, e os desenvolvedores esperam implantar novas versões desses aplicativos várias vezes ao dia. A conteinerização ajuda a empacotar o software para atender a esses objetivos, permitindo que os aplicativos sejam lançados e atualizados de maneira fácil e rápida, sem tempo de inatividade. O Kubernetes ajuda a garantir que esses aplicativos em contêiner sejam executados onde e quando você quiser e os ajuda a encontrar os recursos e ferramentas de que precisam para funcionar. Kubernetes é uma plataforma de código aberto pronta para produção, projetada com a experiência acumulada do Google em orquestração de contêineres, combinada com as melhores idéias da comunidade.</p>
```

File: content/pt/docs/tutorials/kubernetes-basics/_index.html [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+    <div id="basics-modules" class="content__modules">
+      <h2>Módulos básicos do Kubernetes</h2>
+      <div class="row">
+        <div class="col-md-12">
+          <div class="row">
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_01.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/"><h5>1. Crie um cluster Kubernetes</h5></a>
```
Comment (devlware) [inline content/pt/docs/tutorials/kubernetes-basics/_index.html:59]:
```suggestion
                  <a href="/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/"><h5>1. Criar um cluster Kubernetes</h5></a>
```

File: content/pt/docs/tutorials/kubernetes-basics/_index.html [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+                <div class="caption">
+                  <a href="/docs/tutorials/kubernetes-basics/expose/expose-intro/"><h5>4. Exponha seu aplicativo publicamente</h5></a>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/docs/tutorials/kubernetes-basics/scale/scale-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_05.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/docs/tutorials/kubernetes-basics/scale/scale-intro/"><h5>5. Amplie seu aplicativo</h5></a>
```
Comment (devlware) [inline content/pt/docs/tutorials/kubernetes-basics/_index.html:95]:
```suggestion
                  <a href="/docs/tutorials/kubernetes-basics/scale/scale-intro/"><h5>5. Escale seu aplicativo</h5></a>
```

File: content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+                <ul>
+                    <li> Saiba mais sobre implantações de aplicativos. </li>
+                    <li> Implante seu primeiro aplicativo no Kubernetes com o kubectl. </li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h3>Implantações do Kubernetes</h3>
+                <p>
+                Assim que o seu cluster do Kubernetes estiver em execução você pode implementar seu aplicativo em contêiners nele.
```
Comment (devlware) [inline content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:31]:
```suggestion
                Assim que o seu cluster Kubernetes estiver em execução você pode implementar seu aplicativo em contêiners nele.
```

File: content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+                Para fazer isso, você precisa criar uma configuração do tipo <b> Deployment </b> do Kubernetes. O Deployment define como criar e 
+                atualizar instâncias do seu aplicativo. Depois de criar um Deployment, o Master do Kubernetes
```
Comment (devlware) [inline content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:33]: Here, I believe you need to explain that the master node schedules the creation of the pod. I'm not sure if we should keep Master Node in English or say Nó Master or even Control Plane.

File: content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+
+                <p>Você pode criar e gerenciar uma implantação usando a interface de linha de comando do Kubernetes, <b> Kubectl </b>. 
+                    O Kubectl usa a API Kubernetes para interagir com o cluster. Neste módulo, você aprenderá os comandos Kubectl mais comuns necessários para criar implantações que executam seus aplicativos em um cluster Kubernetes.</p>
+
+                <p>Quando você cria um Deployment, você precisará especificar a imagem do contêiner para seu aplicativo e o número de réplicas que deseja executar.
```
Comment (devlware) [inline content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:82]:
```suggestion
                <p>Quando você cria um Deployment, você precisa especificar a imagem do contêiner para seu aplicativo e o número de réplicas que deseja executar. 
```

**Review by devlware — COMMENTED** [review]
Thanks for your work Felipe!
I left some suggestions and one comment that IMO can help the reader.

File: content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html (hunk 9 above)
Comment (femrtnz) [reply content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:33]: Yep, `Nó Master` looks good to me

**Comment by femrtnz** [comment]
Thanks a lot @devlware 

Feel free to add the tags if you feel it is ok then :)

**Pull request merged**